src/app.rs

```diff
@@ -254,12 +268,66 @@ fn parse_string_to_f64(string: String) -> f64 {
     float
 }
 
+fn move_to(
+    app_mode: AppMode,
+    click_position: ClickPosition,
+    click_coord: (f64, f64),
+    is_moving_humanlike: bool,
+    start_coords: (f64, f64),
+    movement_delay_in_ms: u64,
+    mut rng_thread: ThreadRng,
+) {
+    if app_mode == AppMode::Humanlike {
+        // Move mouse slowly to saved coordinates if requested
+        if click_position == ClickPosition::Coord && is_moving_humanlike {
+            let mut current_x = start_coords.0;
+            let mut current_y = start_coords.1;
+            for _n in 0..=5 {
```

Comment:
Can we handle the full cursor motion here please?

From what I understand, currently, the max motion is of 50px and then we'll have to wait until the next `update()` loop

Reply:
> Can we handle the full cursor motion here please?
> 
> From what I understand, currently, the max motion is of 50px and then we'll have to wait until the next `update()` loop

What do you mean by "full cursor motion" ? That - in case the user decides to use a very short movement delay - we're still moving at a maximum of 50 px per cycle?
(that's true, I hadn't thought of that)

Reply:
Looks resolved in the latest! Thanks